src/Symfony/Component/Console/Application.php

```diff
@@ -325,6 +325,9 @@ public function getLongVersion()
         if ('UNKNOWN' !== $this->getName() && 'UNKNOWN' !== $this->getVersion()) {
             return sprintf('<info>%s</info> version <comment>%s</comment>', $this->getName(), $this->getVersion());
         }
+        if ('UNKNOWN' !== $this->getName()) {
+            return sprintf('<info>%s</info>', $this->getName());
+        }
```

Comment:
It's what I had done at first but I thought I was less clear. Anyway I can change no problems !